pkgs/development/tools/analysis/clazy/default.nix

```
@@ -28,6 +28,11 @@ stdenv.mkDerivation rec {
     makeWrapper
   ];
 
+  postPatch = ''
+    # clazy expects brew readlink on macOS, but we have the GNU version
+    substituteInPlace CMakeLists.txt --replace-fail greadlink readlink
```

Maybe this should refer to `coreutils` as well

```suggestion
    substituteInPlace CMakeLists.txt --replace-fail greadlink ${coreutils}/bin/readlink
```